.gitignore

```diff
@@ -11,16 +8,27 @@ loader/
 log.txt
 *.sublime-project
 *.sublime-workspace
+.DS_Store
+Thumbs.db
+UserInterfaceState.xcuserstate
+.env
 
-.stencil/
+# Manifold
+.manifold.yml
+
+# Stencil
+!www/favicon.ico
```

Comment:
I don’t know why this is excluded. This should come after `www` but again you don’t really need to commit this so I’d just say remove it